BitwardenShared/UI/Vault/ImportLogins/ImportLogins/ImportLoginsView.swift

```diff
@@ -134,7 +134,7 @@ struct ImportLoginsView: View {
             .tint(Asset.Colors.textInteraction.swiftUIColor)
 
             VStack(spacing: 12) {
-                AsyncButton(Localizations.continue) {
+                AsyncButton(step == totalSteps ? Localizations.done : Localizations.continue) {
```

Comment:
should we add a test for tapping "Done" and a snapshot test as well?

Reply:
Updated!